internal/rbd/controllerserver.go

```diff
@@ -244,6 +244,17 @@ func (cs *ControllerServer) parseVolCreateRequest(
 }
 
 func (rbdVol *rbdVolume) ToCSI(ctx context.Context) (*csi.Volume, error) {
+	switch {
+	case rbdVol.VolID == "":
+		return nil, fmt.Errorf("%q may not have an empty volume-id", rbdVol)
```

Comment:
need to replace the error statement? because it says `may not`  it should be `may have`

Reply:
Well, the error is that the rbdVolume has an empty volume-id, and that is not allowed. Do you want it like this?
```go
fmt.Errorf("%q has an empty volume-id", rbdVol)
```
To me the meaning is the same, but if the 2nd is clearer, I'm happy to change it.

Reply:
make sense, but `may not have an empty volume-id` should be `may have an empty volume-id` we should remove `not` from it , isnt it?

Reply:
The error would read like `replicated/image may not have an empty volume-id` when the volume-id empty.

I think it is strange to report an error like `replicated/image may have an empty volume-id` when the volume-id is, in fact, empty. So, it actually _may not have_ an empty volume-id (why return an error?).

Maybe this is better?
```go
fmt.Errorf("%q does not have a volume-id set", rbdVol)
```

Reply:
sounds good 👍🏻

Reply:
done, please check again